packages/jest-runtime/src/index.ts

```diff
@@ -967,16 +974,17 @@ class Runtime {
 
   private _execModule(
     localModule: InitialModule,
+    runtimeModuleSource: string | undefined,
     options: InternalModuleOptions | undefined,
     moduleRegistry: ModuleRegistry,
     from: Config.Path | null,
-  ) {
+  ): any | undefined {
```

Comment:
```ts
  private _execModule<T = unknown>(
    localModule: InitialModule,
    runtimeModuleSource: string | undefined,
    options: InternalModuleOptions | undefined,
    moduleRegistry: ModuleRegistry,
    from: Config.Path | null,
  ) {
  ): T | undefined {
```

Reply:
This gets me an error from `tsc` -
```
packages/jest-runtime/src/index.ts:1062:7 - error TS2322: Type 'unknown' is not assignable to type 'T | undefined'.
  Type 'unknown' is not assignable to type 'T'.
    'T' could be instantiated with an arbitrary type which could be unrelated to 'unknown'.

1062       return compiledFunction.call(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1063         localModule.exports,
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ... 
1079         }),
     ~~~~~~~~~~~
1080       );
     ~~~~~~~~
```
What do you recommend here?

Reply:
do ` as T` at the 1080 line